common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1209,7 +1209,8 @@ def test_get_problem_html(self):
 
         context = render_args[1]
         self.assertEqual(context['problem']['html'], "<div>Test Problem HTML</div>")
-        self.assertEqual(bool(context['check_button']), show_check_button)
+        ## comment out line below because submit button is always present but with enabled/disabled state
+        # self.assertEqual(bool(context['submit_button']), enable_submit_button)
```

Comment:
Don't leave the commented out line. If it makes sense, test for the button enabled/disabled state (I assume it is present in the context). Or if that is sufficiently tested elsewhere, just delete this.